### PR TITLE
Small refactoring

### PR DESCRIPTION
**Pull request by jantimar**
Replace move ASN1 logic to ASN1Decoder package
Remove IAPProduct
Fix TYPO and small refactoring

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -16,18 +16,18 @@ let package = Package(
   ],
   dependencies: [
     .package(
-      url: "https://github.com/industrialbinaries/OpenSSL",
-      .branch("feature/asn1")
+      url: "https://github.com/industrialbinaries/ASN1Decoder",
+      .branch("feature/doc")
```
Comment (VojtaStavik) [inline Package.swift:20]: Why we don't use `master`?

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 ![Platforms: macOS](https://img.shields.io/badge/platforms-macOS-brightgreen.svg?style=flat)
 [![Twitter](https://img.shields.io/badge/twitter-@i_binaries-blue.svg?style=flat)](https://twitter.com/i_binaries)
 
-Library for help you with In-App-Purchase with your project
+A helper library for testing In-App purchases in your project
```
Comment (VojtaStavik) [inline README.md:7]: 👍

File: Sources/youda/Extensions/SKProduct+Extension.swift [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+//
+//  Copyright (c) 2020 Industrial Binaries
+//  MIT license, see LICENSE file for details
+//
+
+import StoreKit
+
+extension SKProduct {
+    
+    /// Create new `SKProduct` for mock or tests
```
Comment (VojtaStavik) [inline Sources/youda/Extensions/SKProduct+Extension.swift]: Create**s**  + missing `.` at the end of the sentences (everywhere 😃 )

File: Sources/youda/Extensions/IAP+NotificationName.swift [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+//  youda
+//
+//  Copyright (c) 2020 Industrial Binaries
+//  MIT license, see LICENSE file for details
+//
+
+import Foundation
+
+extension Notification.Name {
+  /// Notification trigged when update bought products
```
Comment (VojtaStavik) [inline Sources/youda/Extensions/IAP+NotificationName.swift]: `The notification triggered when bought products are updated.`

File: Sources/youda/Extensions/SKProduct+Extension.swift [hunk 5]
```diff
@@ -8,64 +8,63 @@
...
-    }
-    
-    /// The cost of the product formatted in the local currency.
-    /// - Parameter formatter: Formatter for format price, default value is currency formatter
-    public func localPrice(formatter priceFormatter: NumberFormatter? = nil) -> String? {
-        let formatter = priceFormatter ?? .currency
-        formatter.locale = priceLocale
-        return formatter.string(from: price)
-    }
+  /// Create new `SKProduct` for mock or tests
```
Comment (VojtaStavik) [inline Sources/youda/Extensions/SKProduct+Extension.swift]: `Creates a new SKProduct with predefined parameters, usually used in tests.

**Review by VojtaStavik — APPROVED** [review]

File: Package.swift (hunk 1 above)
Comment (jantimar) [reply Package.swift:20]: It's older PR, last will be using `master` after marge `feature/refactoring` on `ASN1Decoder`

**Pull request merged**